docs/design/2020-07-07-change-column-types.md

```diff
@@ -69,6 +69,7 @@ This requires changing the state of `colA` and `idxA` to `StatePublic`, and then
 ### Compatibility issues with MySQL
 
 * Considering the complexity of supporting clustered-index and currently TiDB does not fully support clustered-index, so temporarily this feature does not support type modification for columns with the primary key.
+* In addition, considering the complexity and relevance of some feature support, we temporarily do not support the three features of partition table, generated column, and expression index in the first implementation.
```

Comment:
In addition, given the complexity and relevance of some features, we temporarily do not support the column-type-change on partition table, generated column, and expression index in the first stage of implementation.